src/main/java/com/wavefront/sdk/proxy/WavefrontProxyClient.java

```diff
@@ -290,6 +320,37 @@ public void sendMetric(String name, double value, @Nullable Long timestamp,
     }
   }
 
+  @Override
+  public void sendLog(String name, double value, Long timestamp, String source, Map<String, String> tags)
```

Comment:
this is a deprecated class that will be deleted soon, so just throw instead of supporting impl
throw new IllegalArgumentException("Sending logs using this method is not allowed");